### PR TITLE
Added connection settings for redis-cluster

### PR DESCRIPTION
**Pull request by sunny316**
Added connection settings for redis-cluster

### CODE REVIEW

File: redbeat/schedulers.py [hunk 1]
```diff
@@ -175,6 +179,9 @@ def __init__(self, app=None):
         self.lock_timeout = self.either_or('redbeat_lock_timeout', None)
         self.redis_url = self.either_or('redbeat_redis_url', app.conf['BROKER_URL'])
         self.redis_use_ssl = self.either_or('redbeat_redis_use_ssl', app.conf['BROKER_USE_SSL'])
+        self.redbeat_redis_cluster_options = self.either_or('redbeat_redis_cluster_options', {
+            'startup_nodes': [{"host": "localhost", "port": "30001"}]
```
Comment (sibson) [inline redbeat/schedulers.py]: Is `localhost:30001` the right default for most people?

**Comment by sibson** [comment]
I also wonder if we should nest the configuration under ```BROKER_TRANSPORT_OPTIONS``` like was done for [sentinel support](https://redbeat.readthedocs.io/en/latest/config.html#sentinel-support)

File: redbeat/schedulers.py (hunk 1 above)
Comment (sunny316) [reply redbeat/schedulers.py]: In their dev, yes. 
https://redis.io/topics/cluster-tutorial This is the url followed to start with redis cluster. The default port when using their create-cluster script is 30001.
https://redis.io/topics/cluster-tutorial#creating-a-redis-cluster-using-the-create-cluster-script

**Comment by sunny316** [comment]
All the requested changes have been made. Also while updating the configuration docs, I realised a couple of configuration options under sentinel which are never being used in redbeat context namely CELERY_RESULT_BACKEND_TRANSPORT_OPTIONS and CELERY_RESULT_BACKEND

**Comment by sunny316** [comment]
Hi Sibson, thanks for approving the changes and the merge. 
I wanted to know When will you be drafting a release with the new code ?

**Pull request merged**